src/platform/site-wide/side-nav/tests/components/NavItem.unit.spec.js

```diff
@@ -34,7 +34,7 @@ describe('<NavItem>', () => {
       index: 1,
       renderChildItems: noop,
       sortedNavItems: [item],
-      toggleItemExpanded: noop,
+      trackEvents: noop,
```

Comment:
Should there be a unit test that confirms `trackEvents` is called when the items is clicked?

Reply:
Probably a good a idea, but we might need to avoid the GA events to be recorded on the test.

Reply:
Here's a test, with a bit of refactoring: https://github.com/department-of-veterans-affairs/vets-website/compare/vamc-issue-9943...vamc-issue-9943-mps

The `collect` requests from localhost are being rejected by Google, so I'm pretty sure we don't need to worry about spamming GA.